spec.html

```diff
@@ -27890,16 +27891,11 @@ <h1>Properties of the String Constructor</h1>
         <h1>String.fromCharCode ( ..._codeUnits_ )</h1>
         <p>The `String.fromCharCode` function may be called with any number of arguments which form the rest parameter _codeUnits_. The following steps are taken:</p>
         <emu-alg>
-          1. Let _codeUnits_ be a List containing the arguments passed to this function.
-          1. Let _length_ be the number of elements in _codeUnits_.
           1. Let _elements_ be a new empty List.
-          1. Let _nextIndex_ be 0.
-          1. Repeat, while _nextIndex_ &lt; _length_
-            1. Let _next_ be _codeUnits_[_nextIndex_].
+          1. For each element _next_ of _codeUnits_, in ascending index order, do
```

Comment:
Converting the 'while' loop to a 'for' loop (and similarly below) appears to be orthogonal to the main point of this PR. You might want to pull it out to a separate commit, if not a separate PR.